ProjectLighthouse.Servers.Website/Pages/UserPage.cshtml

```diff
@@ -106,172 +113,199 @@
     {
         <br/>
     }
-    <div class="eight wide column">
-        <div class="ui blue segment">
-            <h2>@Model.Translate(ProfileStrings.Biography)</h2>
-            @if (string.IsNullOrWhiteSpace(Model.ProfileUser.Biography))
-            {
-                <p>@Model.Translate(ProfileStrings.NoBiography, Model.ProfileUser.Username)</p>
-            }
-            else
-            {
-                <p style="overflow-wrap: anywhere;">@HttpUtility.HtmlDecode(Model.ProfileUser.Biography)</p>
-            }
-        </div>
-    </div>
-    @if (isMobile)
+    @if (!Model.ProfilePrivate)
     {
-        <br/>
-    }
-    <div class="eight wide column">
-        <div class="ui red segment">
-            <h2>@Model.Translate(GeneralStrings.RecentActivity)</h2>
-            <p>@Model.Translate(GeneralStrings.Soon)</p>
+        <div class="eight wide column">
+            <div class="ui blue segment">
+                <h2>@Model.Translate(ProfileStrings.Biography)</h2>
+                @if (string.IsNullOrWhiteSpace(Model.ProfileUser.Biography))
+                {
+                    <p>@Model.Translate(ProfileStrings.NoBiography, Model.ProfileUser.Username)</p>
+                }
+                else
+                {
+                    <p style="overflow-wrap: anywhere;">@HttpUtility.HtmlDecode(Model.ProfileUser.Biography)</p>
+                }
+            </div>
         </div>
-    </div>
-    @if (isMobile)
-    {
-        <br/>
+        @if (isMobile)
+        {
+            <br/>
+        }
+        <div class="eight wide column">
+            <div class="ui red segment">
+                <h2>@Model.Translate(GeneralStrings.RecentActivity)</h2>
+                <p>@Model.Translate(GeneralStrings.Soon)</p>
+            </div>
+        </div>
+        @if (isMobile)
+        {
+            <br/>
+        }    
     }
 </div>
 
-<div class="ui grid">
-    @{
```

Comment:
probably out the scope of this PR but UI changes should NOT be done on the serverside, there's a whole language for this, it's called CSS (this is more as a general thing for lighthouse as a whole, but this here is a really good example of why)

Reply:
I believe slendy is working on a solution for this, not sure